rapids-cmake/cpm/versions.json

```
@@ -5,11 +5,16 @@
       "git_url" : "https://github.com/google/benchmark.git",
       "git_tag" : "v${version}"
     },
+    "CCCL" : {
+      "version" : "2.2.0",
+      "git_url" : "https://github.com/NVIDIA/cccl.git",
+      "git_tag" : "v${version}"
+    },
     "cuco" : {
       "version" : "0.0.1",
       "git_shallow" : false,
-      "git_url" : "https://github.com/NVIDIA/cuCollections.git",
-      "git_tag" : "7c76a124df0c2cd3fd66e3e080b9470a3b4707c6"
+      "git_url" : "https://github.com/bdice/cuCollections.git",
+      "git_tag" : "cccl-2.2.0"
```

Reverting cuCollections changes so we can merge this.
```suggestion
      "git_url" : "https://github.com/NVIDIA/cuCollections.git",
      "git_tag" : "7c76a124df0c2cd3fd66e3e080b9470a3b4707c6"
```